src/InputWithLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import { useRef } from "react";
+
+function InputWithLabel({ todoTitle, handleTitleChange }, props) {
```

Comment:
This won't work since the first object in parentheses is the component `props`, so the additional one will be empty. Instead, since you just want the `children`, add the explicitly to the object

Reply:
To see this, you'll noticed right now the "Title" label isn't showing up next to the input